spec/system/view_pages_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
```

Comment:
We've used `spec/features/` on the other apps so we should probably standardise to that.

Reply:
agreed, done https://github.com/DFE-Digital/register-trainee-teacher-data/pull/8/commits/c861f6aa8b7f3f8592200da2691fb542a5363e44